apps-devstg/us-east-1/alb-logs-s3-bucket/bucket.tf

```diff
@@ -0,0 +1,56 @@
+#
+# Create an S3 bucket for storing ALB access logs
+#
+module "s3_bucket_alb_logs" {
+  source = "github.com/binbashar/terraform-aws-s3-bucket.git?ref=v3.15.2"
+
+  bucket = "${local.bucket_name}-logs"
+  acl    = "log-delivery-write"
+  # Allow deletion of non-empty bucket
+  force_destroy = true
+
+  control_object_ownership = true
+  object_ownership         = "ObjectWriter"
+
+  attach_elb_log_delivery_policy = true # Required for ALB logs
+  attach_lb_log_delivery_policy  = true # Required for ALB/NLB logs
+
+  lifecycle_rule = [
+    {
+      id      = "billing-objects-logs"
+      enabled = true
+      prefix  = "logs/"
```

Comment:
How about other prefixes? I'd recommend setting a default rule for them so that objects outside the "logs" prefix are also covered.

Reply:
I agree. Prefix will be managed in the source, and we want to have the same rule for all.